source/common/common/fancy_logger.h

```diff
@@ -0,0 +1,107 @@
+#pragma once
+
+#include <string>
+
+#include "absl/container/flat_hash_map.h"
+#include "absl/synchronization/mutex.h"
+#include "spdlog/spdlog.h"
+
+namespace Envoy {
+
+using FancyMap = absl::flat_hash_map<std::string, std::shared_ptr<spdlog::logger>>;
+using FancyMapPtr = std::shared_ptr<FancyMap>;
+using SpdLoggerPtr = std::shared_ptr<spdlog::logger>;
```

Comment:
nit: move this before FancyMap thenyou can use it there
not: per style guide this is SpdLoggerSharedPtr

Reply:
Addressed now.